recipes-core/packagegroups/packagegroup-xenclient-dom0.bb

```diff
@@ -110,6 +110,8 @@ RDEPENDS_${PN} = " \
     udev-extraconf-dom0 \
     vglass \
     disman \
+    varstored \
+    varstored-watch \
```

Comment:
With proper RDEPENDS, which I think you have now, this patch https://github.com/OpenXT/xenclient-oe/pull/1434/commits/489d60df208661e91ac2198cad878240881e9b4e can be removed.